Vonk.Plugin.DocumentOperation.Tests/LoggerUtils.cs

```diff
@@ -2,19 +2,26 @@
 using Microsoft.Extensions.Logging;
 using Moq;
 
-namespace Vonk.Plugin.DocumentOperation.Test
+namespace Vonk.UnitTests.Framework.Helpers
```

Comment:
Not sure when and how we make Vonk.UnitTests.Framework public. Sure you want to adjust your namespace to it already?